libs/libc/stdio/lib_fclose.c

```diff
@@ -64,6 +64,7 @@ int fclose(FAR FILE *stream)
   int errcode = EINVAL;
   int ret = ERROR;
   int status;
+  int fd = (int)(intptr_t)stream->fs_cookie;
```

Comment:
move after line 126 since stream may be NULL.

Reply:
done